api/bonsai_api/cli.py

```diff
@@ -1,39 +1,35 @@
 """Commmand line interface to server component."""
 
 import asyncio
-import json
-import random
-import string
+from csv import DictWriter
 from logging import getLogger
+from typing import Callable
 
 import click
 from pymongo.errors import DuplicateKeyError
 
 from .__version__ import VERSION as version
+from .auth import generate_random_pwd
 from .config import USER_ROLES
 from .crud.group import create_group as create_group_in_db
 from .crud.sample import get_sample, get_samples, update_sample
 from .crud.tags import compute_phenotype_tags
 from .crud.user import create_user as create_user_in_db
+from .db import verify
 from .db.index import INDEXES
 from .db.utils import get_db_connection
 from .io import sample_to_kmlims
 from .models.group import GroupInCreate, pred_res_cols
-from .models.sample import SampleInCreate
+from .models.sample import SampleInCreate, SampleInDatabase
 from .models.user import UserInputCreate
 
 LOG = getLogger(__name__)
 
 
-def _generate_random_pwd(length=15) -> str:
-    symbols = string.digits + string.ascii_lowercase + string.ascii_uppercase
-    return "".join(random.sample(symbols, length))
-
-
 @click.group()
 @click.version_option(version)
 @click.pass_context
-def cli(ctx):
+def cli(_):
```

Comment:
An alternative would be `_ctx` to show what it is while showing that it isn't used